src/game/client/components/scoreboard.cpp

```diff
@@ -653,15 +651,7 @@ void CScoreboard::OnRender()
 	if(m_Active)
 		m_pClient->m_pMotd->Clear();
 
-	// if statboard active don't show scoreboard
-	if(m_pClient->m_pStats->IsActive())
-		return;
-
-	if(!Active())
-		return;
-
-	// don't render scoreboard if menu or motd is open
-	if(m_pClient->m_pMenus->IsActive() || m_pClient->m_pMotd->IsActive())
```

Comment:
This would now render the scoreboard when the motd is open.

Reply:
When you open the scoreboard the motd is closed anyway.

https://github.com/teeworlds/teeworlds/blob/master/src/game/client/components/scoreboard.cpp#L652-L654

Reply:
Yeah, but when the scoreboard is opened automatically like on game/round end, death etc. the motd will close and [this issue](https://github.com/teeworlds/teeworlds/issues/1899) opens up again.

Reply:
Ah nevermind there was a follow up fix.